frontend/containers/dashboard/open-calls/table/cells/actions/component.tsx

```diff
@@ -97,24 +100,24 @@ export const CellActions: FC<CellActionsProps> = ({
           <FormattedMessage defaultMessage="Edit" id="wEQDC6" />
         </a>
       </Link>
-      <RowMenu direction="top" onAction={handleRowMenuItemClick}>
+      <RowMenu direction={isFirst ? 'bottom' : 'top'} onAction={handleRowMenuItemClick}>
```

Comment:
I don't think this is an ideal solution, but it's a quick fix to prevent the first entry from being cut off. 

<img width="474" alt="Screenshot 2022-08-24 at 05 04 51" src="https://user-images.githubusercontent.com/6273795/186326656-35228c77-cd0c-46ee-a4f7-983cd389fac7.png">

We've looked into this before (the projects list had a similar issue) and didn't find a solution quickly, (`@react-aria/menu` does have a `shouldFlip` prop, but it was not working) so we ended up "solving it" this way as well.